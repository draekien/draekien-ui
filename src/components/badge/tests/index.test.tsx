import Badge from '..';
import * as React from 'react';
import { render } from '@testing-library/react';

describe('Badge component', () => {
  test('should match snapshot', () => {
    const { container } = render(<Badge>1</Badge>);

    expect(container).toMatchSnapshot();
    expect(container.textContent).toContain('1');
  });
});
