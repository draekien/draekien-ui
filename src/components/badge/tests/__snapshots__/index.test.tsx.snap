// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Badge component should match snapshot 1`] = `
<div>
  <div
    class="css-opfat8-Badge"
  >
    1
  </div>
</div>
`;
